### PR TITLE
Fix the online/offline search

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Fix the location closing and online/offline search
## Impacts

Radius accounting and the admin gui
## NEWS file entries

Bug Fixes
+++++++++
- Remove locationlog closing on radius accounting stop

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Bump

File: sbin/pfmon [hunk 1]
```diff
@@ -252,13 +252,8 @@ sub registertasks  {
...
     register_task(
-        'update node online stats', 
-        'online_node_count_interval', 
-        \&update_node_online_stats
-    );
-    register_task(
-        'fingerbank data update', 
-        'fingerbank_data_update_interval', 
+        'fingerbank data update',
+        'fingerbank_data_update_interval',
```
Comment (louismunro) [inline sbin/pfmon:256]: You're sure this should be in this PR?

**Comment by louismunro** [comment]
On it.
I want to test it before we merge so we don't screw up the maintenance branch.

File: sbin/pfmon (hunk 1 above)
Comment (jrouzierinverse) [inline sbin/pfmon:256]: That is just white space noise

Comment (louismunro) [inline sbin/pfmon:256]: Ah, got it now.
Forget I said anything.

**Pull request merged**